EIPS/eip-695.md

```diff
@@ -0,0 +1,80 @@
+## Preamble
+
+    EIP: 695
+    Title: Create `eth_chainId` method for JSON-RPC
+    Author: Isaac Ardis: isaac.ardis@gmail.com, Wei Tang: hi@that.world, [@tcz001](https://github.com/tcz001)
+    Type: Standard Track
+    Category: Interface
+    Status: Draft
+    Created: 2017-08-21
+
+
+## Simple Summary
+Include `eth_chainId` method in `eth_`-namespaced JSON-RPC methods.
+
+## Abstract
+The `eth_chainId` method should return a single STRING result
+for an integer value in hexadecimal format, describing the
+currently configured "Chain Id" value used for signing replay-protected transactions,
+introduced via EIP-155.
+
+## Motivation
+Currently although we can use net_version RPC call to get the
+current network ID, there's no RPC for querying the chain ID. This
+makes it impossible to determine the current actual blockchain using
+the RPC.
+
+## Specification
+
+----
+
+### eth_chainId
+
+Returns the currently configured chain id, a value used in replay-protected transaction
+signing as introduced by EIP-155.
+
+##### Parameters
+none
+
+##### Returns
+
+`QUANTITY` - big integer of the current chain id. Defaults are mainnet=61, morden=62.
```

Comment:
JSON RPC has String, Number, Booleans or Null.  I guess it's a String representing an integer.

Reply:
Maybe "big integer" is the usual way to say it?

Reply:
Also, `QUANTITY`, for example: https://github.com/ethereum/wiki/wiki/JSON-RPC#net_peercount. 

I'll change it to `integer number of the current chain id. [...]`, ok?

Reply:
What is this `QUANTITY`?

Reply:
> When encoding QUANTITIES (integers, numbers): encode as hex, prefix with "0x", the most compact representation (slight exception: zero should be represented as "0x0"). Examples:
> 
> 0x41 (65 in decimal)
> 0x400 (1024 in decimal)

https://github.com/ethereum/wiki/wiki/JSON-RPC#hex-value-encoding

Reply:
Thank you.  Will you add that link to the PR?  Maybe in a `Reference` section.

Reply:
Done, 1d1799b